pylint/checkers/classes.py

```diff
@@ -957,6 +957,13 @@ def _check_protected_attribute_access(self, node):
                node.expr.func.name == 'super':
                 return
 
+            # If the expression begins with a call to type(self), that's ok.
```

Comment:
Let's not use line continuation with ``\``. I know the current code uses this, but I would prefer the new code to drop it entirely and use parens instead.